src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c

```diff
@@ -1,5 +1,5 @@
 /*
- * File z_en_bigpo.c
+ * File: z_en_bigpo.c
  * Overlay: ovl_En_Bigpi
  * Description: Big Poe. Leader of the Poes, found under Dampe's house and in Beneath the Well
```

Comment:
It was like this when I got here. The spreadsheet actually says "Big Poe" and nothing else. I can strip out the "Leader of the Poes" thing if it's not lore-accurate.

Reply:
Oh lol that was me, also `ovl_En_Bigpi` is wrong, feel free to change it

Reply:
Yeah, he says this specifically, which is what Elliptic was referring to:
![image](https://user-images.githubusercontent.com/12245827/142811550-00b4a8fa-a3b2-429c-8017-f976d1be92c3.png)
I guess that sort of implies he's the leader of the Poes, since there aren't *non*-Poe ghosts in this universe, but it's not explicitly stated.
Thanks for the `bigpi` thing; I'll fix that.